packages/js/ai/src/index.ts

```diff
@@ -4,6 +4,7 @@
 export {
 	useCompletion as __experimentalUseCompletion,
 	UseCompletionError,
+	useBackgroundRemoval,
```

Comment:
We might want to start with prepending this with `experimental` like we're doing above, so we can iterate in the short-medium term.

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/40089/commits/f71ef861fa71066a6dcf3c1bef0ec70a5a58a22f